<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOrto</class>
 <widget class="QWidget" name="CreateOrto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBoxExtraerOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Extreure Orto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteDatesOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete dates</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonFolderOutOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Directori desti</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFolderOutOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="labelAmbitoProyectoOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ambit de proyecte</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAmbitoProyectoOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="labelTamanoPixelOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Tamany pixel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTamanoPixelOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="labelUtmOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>UTM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxUtmOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxFootPrintMaskOrto">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>FootPrintMask</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="labelAnchoPasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Amplada passada</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAnchoPasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="labelOffsetPasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Offset passada</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxOffsetPasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
